### PR TITLE
DFPL-2140 Upload Documents

### PR DESCRIPTION
**Pull request by Braimah101**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-2140

### Change description ###

Objective: Extend the smoke test coverage by introducing a dedicated section to assess the functionality and reliability of the document upload feature within the application.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: playwright-e2e/fixtures/create-fixture.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { ManageHearings } from "../pages/manage-hearings";
 import { GatekeepingListing } from "../pages/gatekeeping-listing";
 import { CaseDetails } from "../pages/case-details";
 import { RespondentDetails } from "../pages/respondent-details";
+import { UploadDocuments } from "../pages/upload-documents";
```
Comment (Iphiclus) [inline playwright-e2e/fixtures/create-fixture.ts:20]: I believe this has already been added with AddApplicationDocuments page and ticket: https://tools.hmcts.net/jira/browse/DFPL-2150

File: playwright-e2e/pages/start-application.ts [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import { type Page, type Locator, expect } from "@playwright/test";
 
 export class StartApplication {
+  uploadDocumentsHasBeenUpdated() {
```
Comment (Iphiclus) [inline playwright-e2e/pages/start-application.ts:4]: Hi, Is this needed?

File: playwright-e2e/pages/start-application.ts [hunk 3]
```diff
@@ -20,6 +23,7 @@ export class StartApplication {
   readonly uploadDocumentsLink: Locator;
   readonly upLoadDocsInProgress: Locator;
   readonly respondentsDetailsLink: Locator;
+
```
Comment (Iphiclus) [inline playwright-e2e/pages/start-application.ts:26]: Hi, is this newline needed?

File: playwright-e2e/pages/upload-documents.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        await this.page.locator('#temporaryApplicationDocuments_1_document').click();
+        await this.page.locator('#temporaryApplicationDocuments_1_document').setInputFiles(config.testWordFile);
+        await this.page.locator('#temporaryApplicationDocuments_1_confidential-CONFIDENTIAL').check();
+        await this.page.getByLabel('Give details of documents to').click();
+        await this.waitForAllUploadsToBeCompleted();
+        await this.clickContinue();
+        await this.checkYourAnsAndSubmit();
+    }
+}
+
```
Comment (Iphiclus) [inline playwright-e2e/pages/upload-documents.ts:45]: Hi, are these newlines needed?

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 5]
```diff
@@ -34,11 +35,11 @@ test("Smoke Test @smoke-test @accessibility", async ({
   await createCase.submitCase(createCase.generatedCaseName);
   await createCase.checkCaseIsCreated(createCase.generatedCaseName);
 
-  // Orders and directions sought
+  // // Orders and directions sought
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:38]: Hi are the double double backslashes needed?

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 6]
```diff
@@ -34,11 +35,11 @@ test("Smoke Test @smoke-test @accessibility", async ({
...
   await ordersAndDirectionSought.ordersAndDirectionsNeeded();
   await startApplication.addApplicationDetailsHeading.isVisible();
 
-  // Hearing urgency
+  // // Hearing urgency
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:42]: Hi are the double double backslashes needed?

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 7]
```diff
@@ -52,7 +53,7 @@ test("Smoke Test @smoke-test @accessibility", async ({
   await hearingUrgency.saveAndContinueButton.click();
   await startApplication.addApplicationDetailsHeading.isVisible();
 
-  // Grounds for the application
+  // // Grounds for the application
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:56]: Hi are the double double backslashes needed?

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 8]
```diff
@@ -72,7 +73,7 @@ test("Smoke Test @smoke-test @accessibility", async ({
   await addApplicationDocuments.uploadDocumentSmokeTest();
   await startApplication.addApplicationDocumentsInProgress();
 
-  // Add respondents' details
+  // // Add respondents' details
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:76]: Hi are the double double backslashes needed?

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 9]
```diff
@@ -81,15 +82,20 @@ test("Smoke Test @smoke-test @accessibility", async ({
...
+    .analyze();
 
   await testInfo.attach('accessibility-scan-results', {
     body: JSON.stringify(accessibilityScanResults, null, 2),
     contentType: 'application/json'
   });
 
-expect(accessibilityScanResults.violations).toEqual([]);
+  expect(accessibilityScanResults.violations).toEqual([]);
+
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:100]: Hi, is this newline needed?

**Review by Iphiclus — CHANGES_REQUESTED** [review]
I believe this has already been added with AddApplicationDocuments page and ticket: https://tools.hmcts.net/jira/browse/DFPL-2150
But I might be wrong?

**Comment by DanCatchpole** [comment]
ticket was withdrawn

**Pull request closed without merging**